pkg/providers/native/flag.go

```diff
@@ -14,18 +18,31 @@ const createUsageExample = `  autok3s -d create \
 const joinUsageExample = `  autok3s -d join \
     --provider native \
     --ssh-key-path <ssh-key-path> \
-    --ip <existing server ip> \
```

Comment:
why remove `--ip` flag?  Are you going to use '-master-ips' to try to join an existing cluster ？

Reply:
We have saved the native cluster to DB, so we can get `--ip` value from DB. User is no need to specify this parameter when trying to join nodes to this cluster.

Reply:
If so, is the `--name` attribute missing?

Reply:
yes, I will fix it.